### ISSUE TITLE
A user can go through the application speaking french only

### ISSUE THREAD

**Issue by XavierMeunier** [PANDA-Guide/PandaGuideApp#10]
- Every labels should be written in french
- Every accessibility labels read by Voice Over should be in french
